blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button-component/ff_module-dropdown-button-component.md

```diff
@@ -0,0 +1,18 @@
+## React Button Component
+<div data-ff_module-dropdown-button-component=""></div>
+
+See [ff-module-dropdown-button](/blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button.html) for details on dropdown styling and modifiers.
+
+This component just renders an html dropdown button, it doesn't replicate the dropdown functionality which already exists in the core 'vanilla' dropdown file 'ff_module-dropdown.js'.
+
+**The core dropdown file is activated automatically once the component mounts.**
+
+# Props 
+- **text**: {String, required} The text rendered by the button
+- **id**: {String, required} The -target and -trigger id used by the core dropdown js file.
```

Comment:
I'm not sure I understand what this is used for. Is this just an ID that needs to be unique on the page for use just within this component? If so, couldn't you just automatically generate it in the component?